### ISSUE TITLE
Have non space occupying hidden content

### ISSUE THREAD

**Issue by andreasKroepelin** [andreasKroepelin/typst-slides#8]
It should be possible to hide content such that it doesn't occupy any space. I especially imagine a variant of `#one-by-one` that allows to substitute contents by each other.

**Comment by andreasKroepelin** [comment]
Yes, I was thinking of something quite similar. I guess the hardest problem here (as always 😅) is naming. Currently we have for example `#only`, so I'm not sure how to properly differentiate between space occupying and completely gone and still keep concise function names.

**Comment by palainp** [comment]
Another try I had was to add a new state for `cover-mode` like:
```
#let cover-mode-absent = cover-mode.update("absent")

#let slides-custom-hide(body) = {
    locate( loc => {
        let mode = cover-mode.at(loc)
        // wrap in box to avoid hiding issues with list, equation and other types
        if mode == "hide" {
            hide(full-box(body))
        } else if mode == "mute" {
            text(gray.lighten(50%), full-box(body))
        } else if mode == "absent" {
            // produce nothing, the content is simply not-present...
        } else {
            panic("Illegal `cover-mode`: " + mode)
        }
    })
}
```
But I also wanted to have _muted_ items, so this doesn't work for me.

**Comment by andreasKroepelin** [comment]
Resolved by #16
